game.libretro.scummvm/addon.xml

```diff
@@ -13,8 +10,11 @@
                library_windx="game.libretro.scummvm.dll"
                library_wingl="game.libretro.scummvm.dll">
         <platforms></platforms>
-        <extensions></extensions>
+        <extensions>scummvm|scum</extensions>
         <supports_vfs>false</supports_vfs>
+        <supports_game_loop>true</supports_game_loop>
+        <supports_standalone>true</supports_standalone>
```

Comment:
Oh, I fear I didn't test this carefully enough. When standalone is set, it will always launch in standalone mode. For this to work, I guess we'd have to modify the check here https://github.com/garbear/xbmc/blob/retroplayer-17alpha1/xbmc/games/addons/GameClient.cpp#L254 and test if the given file has an extension that is supported by the addon. Would that work?